addon/ng2/commands/github-pages-deploy.ts

```diff
@@ -52,8 +53,16 @@ module.exports = Command.extend({
       type: String,
       default: '',
       description: 'Github username'
+    }, {
+      name: 'output-path',
+      type: String,
+      default: 'dist/',
```

Comment:
just another thought on this if we remove it there will not be a default value for *outputPath* if it's not set in the config file. I can handle this in `CommandHelper.loadDefaults` by making sure *outputPath* has a value. 

Do you have any issues if I do it this way?